src/conversation.js

```diff
@@ -25,7 +26,7 @@ const shutdown = (bot, message) => {
       {
         pattern: bot.utterances.yes,
         callback: function(response, convo) {
-          convo.say("Alright Sir, if you'll not be needing me, I'll close down for awhile");
+          convo.say('Alright Sir, if you\'ll not be needing me, I\'ll close down for awhile');
```

Comment:
I'd prefer backtick and no apostrophe escape here. I hate escapes. I don't have any good reason.